packages/react/src/components/fieldset/Fieldset.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import { Fieldset } from './Fieldset';
+import { TextInput } from '../textInput/TextInput';
+import { RadioButton } from '../radioButton/RadioButton';
+import { SelectionGroup } from '../selectionGroup/SelectionGroup';
+
+export default {
+  component: Fieldset,
+  title: 'Components/Fieldset',
+  decorators: [withKnobs, (storyFn) => <div style={{ maxWidth: '600px' }}>{storyFn()}</div>],
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {
+    border: false,
+    heading: 'Applicant information',
+  },
+};
+
```

Comment:
Should we add example of disabled fieldset? According to specs in disabled fieldset all the form elements become disabled: https://www.w3schools.com/jsref/prop_fieldset_disabled.asp. 🤔

Reply:
I tested it, and it did disable the inputs, but it did not update the styles. It might be impossible to change this behavior without refactoring the disabled styles of all HDS inputs 🤔